utils/seed.js

```diff
@@ -0,0 +1,36 @@
+import User from '../models/User.js';

```

Comment:
PRAISE: Thank you for doing this. Adding a data seeder to create/delete data can help with testing and take the headache out of making up fake user data with names of characters from sweet 80s movies!